src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustIndividualValidatorTest.java

```diff
@@ -266,36 +268,39 @@ void testErrorReportedWhenDateBecomeInterestedPersonIsInTheFuture() {
         assertError(qualifiedFieldName, validationMessage, errors);
     }
 
-    @Test
-    void testErrorReportedWhenServiceAddressSameAsUsualResidentialAddressFieldIsNull() {
+    @ParameterizedTest
+    @ValueSource(strings = {"true", "false"})
```

Comment:
Apparently you can't pass booleans as a parameter but you can get around it by passing strings of true and false which are then auto converted to booleans to match the test method parameter type.

Reply:
https://junit.org/junit5/docs/5.8.0/api/org.junit.jupiter.params/org/junit/jupiter/params/provider/ValueSource.html